packages/lib/src/vis/shared/Html.tsx

```diff
@@ -27,15 +30,18 @@ function Html(props: Props) {
   const [renderTarget] = useState(() => document.createElement('div'));
 
   useLayoutEffect(() => {
-    ReactDOM.render(<>{children}</>, renderTarget); // eslint-disable-line react/jsx-no-useless-fragment
-  }, [children, renderTarget]);
+    const root = createRoot(renderTarget);
+    setReactRoot(root);
```

Comment:
Since `root.render(<>children</>)` is now in a separate effect, we need a state to pass the `root` over. This leads to a double render on mount, which is not a big deal. I could probably use a ref to avoid this, since the order of the effects is guaranteed, but it felt like a premature optimisation.